ddl/ddl_db_change_test.go

```diff
@@ -292,14 +292,38 @@ type sqlWithErr struct {
 	expectErr error
 }
 
+type expectQuery struct {
+	sql  string
+	rows []string
+}
+
+// https://github.com/pingcap/tidb/pull/6249 fix the following two test cases.
+func (s *testStateChangeSuite) TestWriteOnlyWriteNULL(c *C) {
```

Comment:
s/WriteOnly/WriteReorg?

Reply:
I'm also curious that why the test is not work in WriteOnly state. But it should arise an error in WriteOnly state.

Reply:
So we need to update line320?
I only want to unify these test names.

Reply:
If we expect to cover the case in Write Only state, this name should be OK.